guides/common/modules/proc_restricting-custom-repositories-to-a-specific-os-version-or-architecture-in-project.adoc

```diff
@@ -0,0 +1,23 @@
+[id="Restricting_Custom_Repositories_to_a_Specific_OS_Version_or_Architecture_in_Project{context}"]
+= Restricting Custom Repositories to a Specific Operating System or Architecture in {Project}
+
+You can configure {Project} to make custom repositories available only on hosts with a specific operating system version or architecture.
```

Comment:
We have an attribute for custom repositories: https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-base.adoc#L48

Maybe you can also add one for title and context?

Reply:
From `custom repos` to just `repository`, it changes the whole context/purpose of the section, No?

Reply:
I believe that RH calls everything that is not "repos for RHEL" _custom_. That is, to my understanding, the reason for those attributes. cc @Lennonka

Reply:
Ohk. Learning here. In this case, I shall go with your suggestion :+1:

Reply:
Yes for using `{customrepo}`, that's exactly why those attributes are there. Also we might go with a singular for the whole procedure?

Reply:
@Lennonka Can you explain why you would go with singular?

Reply:
Please review [the latest commit](https://github.com/theforeman/foreman-documentation/pull/2302/commits/87b692285ed07533ab0ee098dbecf7159cc2e49b). The procedure now uses attributes instead of hard-coding "custom repositories". But I redefined the attribute to be displayed as "custom repository" to conform to Jeremy's advice.